Userland/Libraries/LibWeb/HTML/Window.cpp

```diff
@@ -1863,6 +1864,12 @@ JS_DEFINE_NATIVE_FUNCTION(Window::origin_getter)
     return JS::PrimitiveString::create(vm, impl->associated_document().origin().serialize());
 }
 
+JS_DEFINE_NATIVE_FUNCTION(Window::is_secure_context_getter)
+{
+    auto* impl = TRY(impl_from(vm));
+    return JS::Value(is_secure_context(impl->associated_document().relevant_settings_object()));
```

Comment:
also needs a spec comment

```suggestion
    // The isSecureContext getter steps are to return true if this's relevant settings object is a secure context, or false otherwise.
    return JS::Value(is_secure_context(impl->associated_document().relevant_settings_object()));
```